presto-docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -271,3 +271,12 @@ MongoDB collection has the special field ``_id``. The connector tries to follow
 .. note::
 
     Unfortunately, there is no way to represent ``_id`` fields more fancy like ``55b151633864d6438c61a9ce``.
+
+Limitations
+-----------
+
+The following operations are not supported:
+
+- Deleting data with :doc:`/sql/delete`, :doc:`/sql/drop-table`, or similar
+  operations
+- Updating data, which is generally not supported by Presto
```

Comment:
See https://prestosql.io/docs/current/connector/sqlserver.html#sql-server-connector-limitations for example

Reply:
Is that link a good or a bad example? E.g the section title there should not repeat the name of the connector. Also it does not talk about updating data. So I am not sure what you want me to change in this PR. And of fix up that same section in the the sql server connector.

I am also not aware of the specific limitations so if you can enumerate the SQL commands that wont work that might help (provided we want to list them here)

Reply:
Also in the sql server docs the mapping is in the limitations section. That is also wrong.

Reply:
> E.g the section title there should not repeat the name of the connector. 

good catch

> Also it does not talk about updating data.

because it does not make sense to document a global limitation in each connector's docs, does it?

Reply:
Well.. I think we need a section somewhere that says that Presto does not support updating data in general. Where would we put that? 

And then maybe a connector overview page that speaks about this as well or refers to it. I can work on that. 

And for now I guess I can remove it here .. although it is confusing for users as you can see from the ticket that initiated this work.

Reply:
https://github.com/prestosql/presto/pull/4569

Reply:
Connector overview and other updates have to wait..